app/serializers/search/nested_user_serializer.rb

```diff
@@ -0,0 +1,7 @@
+module Search
+  class NestedUserSerializer
+    include FastJsonapi::ObjectSerializer
+
+    attributes :id, :name, :pro, :profile_image_90, :username
+  end
+end
```

Comment:
We also have the `ClassifiedListingAuthorSerializer` as well. I made a [note](https://github.com/thepracticaldev/dev.to/pull/6132/files#r380384999) in the past that we may want to consider consolidating somehow or at least keep in mind before we end up with many serializers doing something so similar. This is only the second use I've seen so just getting in the habit of reminding us 🙃

Reply:
I figured we would replace that one but didnt want to do it in this PR